bcda/main.go

```diff
@@ -326,6 +326,29 @@ func setUpApp() *cli.App {
 				return err
 			},
 		},
+		{
+			Name:     "delete-dir-contents",
+			Category: "Cleanup",
+			Usage:    "Delete the files in a directory specified by the passed Environment Variable",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "envvar",
+					Usage:       "Name of Env Var that determines the dir to delete",
+					Destination: &envVar,
+				},
+			},
+			Action: func(c *cli.Context) error {
+				dirToDelete := os.Getenv(envVar)
```

Comment:
We could pass in any env var here.  Should we make sure its truly a filepath?

Reply:
added a check to ensure it is a dir before moving on.

Reply:
I guess nothing is stopping me from defining my own env var and executing this like:
```
ROOT_DIR=/ bcda delete-dir-contents
```

So now I'm second guessing the approach of requiring that an env var be passed in.  And I think if we keep it open to _delete any dir_ we could run into accidental issues (not protecting ourselves from accidental file system corruption).  

What is the downside of making this CLI very specific like `delete-cclf-files`, and deleting only the files in that directory (which is specified via env var)?

Reply:
I took our conversation to heart and refactored to accept a directory instead of an env var.